tests/browser/js/PanelTestUtils.js

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2013-2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+/* global fluid, jqUnit */
+"use strict";
+
+// This file is copied from Infusion 2.0's preference editor tests, because it is not included in the npm package
```

Comment:
Do we have a JIRA for this problem? Surely the npm package should include everything which is necessary to run and test Infusion

Reply:
yes, i filed a JIRA for it ( https://issues.fluidproject.org/browse/FLUID-6170 ). I'll update the file to include a reference to that JIRA.

Reply:
I added a link to the JIRA under the comment.